sub_workflows/kf_cnv_somatic_pair_wf.cwl

```diff
@@ -0,0 +1,664 @@
+cwlVersion: v1.1
+class: Workflow
+id: kf_cnv_somatic_pair_wf
```

Comment:
maybe gatk should still be in the id

Reply:
changed the id and renamed the file/reference in the workflow